src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    static final int SIZE_ARRAY = 5;
```

Comment:
```suggestion
    private static final int SIZE_ARRAY = 5;
```

don't forget about access modifiers